tests/admin_checks/tests.py

```diff
@@ -157,18 +162,25 @@ def test_middleware_dependencies(self):
         errors = admin.checks.check_dependencies()
         expected = [
             checks.Error(
-                "'django.contrib.auth.middleware.AuthenticationMiddleware' "
+                "'django.contrib.auth.middleware.AuthenticationMiddleware' (or a subclass) "
                 "must be in MIDDLEWARE in order to use the admin application.",
                 id='admin.E408',
             ),
             checks.Error(
-                "'django.contrib.messages.middleware.MessageMiddleware' "
+                "'django.contrib.messages.middleware.MessageMiddleware' (or a subclass) "
                 "must be in MIDDLEWARE in order to use the admin application.",
                 id='admin.E409',
             )
         ]
         self.assertEqual(errors, expected)
 
+    @override_settings(MIDDLEWARE=[
+        'admin_checks.tests.MyAuthMiddleware',
+        'django.contrib.messages.middleware.MessageMiddleware',
```

Comment:
Do we not need a `MessageMiddleware` subclass here to make sure that path is tested too?